pkg/vmimage/data/script.sh

```diff
@@ -164,7 +172,29 @@ yum -y install \
     tree \
     find \
     WALinuxAgent \
-    yum-utils
+    yum-utils \
+    azure-mdsd \
+    azure-security \
+    azsec-monitor \
+    td-agent
+
+# td-agent needs access to the journal
+usermod -a -G systemd-journal td-agent
+
+# openshift-audit dir ends up with wrong permissions, preemptively fix that
+mkdir -pv /var/log/openshift-audit
+chmod -v 0755 /var/log/openshift-audit
+
+# we need to build gems
+yum -y groups install "Development Tools"
+
+# install the latest stable for td-agent3
+/opt/td-agent/embedded/bin/fluent-gem install fluent-plugin-systemd -v 1.0.1
```

Comment:
Why does fluent ship its own copy of rubygems!? It's also very frustrating that `fluent-plugin-systemd` doesn't ship binary gems because then this whole song and dance could be avoided.

The latest version is 1.0.2, does MS have a hard requirement on 1.0.1?

Reply:
that's ruby's "omnibus" pattern, which means ship all the things.